libioc/Config/Jail/Defaults.py

```diff
@@ -85,22 +103,38 @@
     "exec_prestop": None,
     "exec_stop": "/bin/sh /etc/rc.shutdown",
     "exec_poststop": None,
+    "exec_system_user": False,
```

Comment:
This parameter exists in the iocage json code, but is not used at all. The user executing system commands can be changed by executing the process as the target user.

Reply:
OK! this parameter changes from `False` to `root`.(Because default setting, maybe here is should root user)